pkg/hooks/launch.go

```diff
@@ -261,7 +262,7 @@ func (h *Hook) LaunchUserApplication(appCmd, appContainer, appNetwork string, De
 	}
 }
 
-func (h *Hook) processDockerEnv(appCmd, appContainer, appNetwork string, buildDelay time.Duration) error {
+func (h *Hook) processDockerEnv(appCmd, appContainer, appNetwork string, buildDelay time.Duration, isDockerComposeCmd bool) error {
```

Comment:
why do we need this boolean, can't we just check if the containers are running and then remove them if needed.